samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -33,6 +33,11 @@ The sample performs the following operations:
 The sample also provides a UART overlay that will allow your sample to transfer at a speed of 1M baud, and it enables support for the ``fmfu_mgmt`` command group.
 See :ref:`lib_fmfu_mgmt` for more details.
 
+.. note::
+   UART is the only transport method supported.
+   RTT is enabled by default to ensure the log generated by the sample or other subsystems can be viewed.
```

Comment:
```suggestion
   RTT is enabled by default to ensure that a user can view the log output generated by the sample or other subsystems.
```
Just a suggestion, not sure if it really made it better.

Reply:
Rephrased a bit but kept the suggested structure, thanks!